src/dymaptic.GeoBlazor.Core/Components/Layers/Layer.cs

```diff
@@ -191,7 +191,7 @@ internal override void ValidateRequiredChildren()
     /// </param>
     internal virtual Task UpdateFromJavaScript(Layer renderedLayer)
     {
-        if (renderedLayer.FullExtent is not null)
+        if (renderedLayer?.FullExtent is not null)
```

Comment:
this `?` should not be necessary based on the non-null nature of the method parameter. However, if `null` _is_ sneaking in here from JS, we should back up and check that before this method is called.

Reply:
This was corrected by adding "wcs" as a layer type to the layerConverter in `layer.cs`